packages/blocks/src/store/selectors.js

```diff
@@ -112,6 +202,38 @@ export const getBlockVariations = createSelector(
  * @param {Object}                attributes Block attributes used to determine active variation.
  * @param {WPBlockVariationScope} [scope]    Block variation scope name.
  *
+ * @example
+ * ```js
+ * import { __ } from '@wordpress/i18n';
+ * import { store as blocksStore } from '@wordpress/blocks';
+ * import { store as blockEditorStore } from '@wordpress/block-editor';
+ * import { useSelect } from '@wordpress/data';
+ *
+ * const ExampleComponent = () => {
+ *     // This example assumes that a core/embed block is the first block in the Block Editor.
+ *     const activeBlockVariation = useSelect( ( select ) => {
+ *         // Retrieve the list of blocks.
+ *         const [ firstBlock ] = select( blockEditorStore ).getBlocks();
+ *         // Retrieve the block attributes for the first block.
+ *         const firstBlockAttributes = select(
+ *             blockEditorStore
+ *         ).getBlockAttributes( firstBlock.clientId );
+ *
+ *         // Return the active block variation for the first block.
+ *         return select( blocksStore ).getActiveBlockVariation(
+ *             'core/embed',
```

Comment:
How about also using `firstBlock` for consistency?:
```suggestion
 *             firstBlock.name,
```

Reply:
I've simplified the whole example based on your suggestion - thanks!